src/store/applyMiddleware.js

```diff
@@ -0,0 +1,36 @@
+// Function taken from redux source
+// https://github.com/reactjs/redux/blob/master/src/compose.js
```

Comment:
Thank you for the comment!

Reply:
No problem! Wanted to make sure sources were clear.